src/parser.rs

```diff
@@ -849,6 +851,56 @@ impl<'a> Parser<'a> {
         })
     }
 
+    pub fn parse_ceil_expr(&mut self) -> Result<Expr, ParserError> {
+        self.expect_token(&Token::LParen)?;
+        let expr = self.parse_expr()?;
+        let keyword_to = self.expect_keyword(Keyword::TO);
```

Comment:
Could you use `self.consume_token` here rather than `expect_token`, as a lack of a `TO` isn't really an error?

https://github.com/sqlparser-rs/sqlparser-rs/blob/main/src/parser.rs#L1743